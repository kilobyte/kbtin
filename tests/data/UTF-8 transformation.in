#hexord x abcąęśćdefабцдефгхийאבגדァアィイ;#showme {$x}
#chr x U+20027 U+20000 U+20001 U+20002;#hexord x $x;#showme {$x}
#var i 1;#loop 1,20 {#math i $i*2;#math l $i-1;#math r $i+1;#chr x $l $i $r;#hexord x {$x};#show $x}
#show last test
#chr x U+10ffff;#hexord x $x;#showme {$x}
