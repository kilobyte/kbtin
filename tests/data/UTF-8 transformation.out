U+0061 U+0062 U+0063 U+0105 U+0119 U+015B U+0107 U+0064 U+0065 U+0066 U+0430 U+0431 U+0446 U+0434 U+0435 U+0444 U+0433 U+0445 U+0438 U+0439 U+05D0 U+05D1 U+05D2 U+05D3 U+30A1 U+30A2 U+30A3 U+30A4
U+20027 U+20000 U+20001 U+20002
U+0001 U+0002 U+0003
U+0003 U+0004 U+0005
U+0007 U+0008 U+0009
U+000F U+0010 U+0011
U+001F U+0020 U+0021
U+003F U+0040 U+0041
U+007F U+FFFD U+FFFD
U+00FF U+0100 U+0101
U+01FF U+0200 U+0201
U+03FF U+0400 U+0401
U+07FF U+0800 U+0801
U+0FFF U+1000 U+1001
U+1FFF U+2000 U+2001
U+3FFF U+4000 U+4001
U+7FFF U+8000 U+8001
U+FFFF U+10000 U+10001
U+1FFFF U+20000 U+20001
U+3FFFF U+40000 U+40001
U+7FFFF U+80000 U+80001
U+FFFFF U+100000 U+100001
last test
U+10FFFF
